src/main/java/org/folio/dew/controller/BulkEditController.java

```diff
@@ -160,6 +160,7 @@ public ResponseEntity<UserCollection> postUserContentUpdates(@ApiParam(value = "
         .toJobParameters());
     }
     var updatesResult = userContentUpdateService.process(jobCommand, contentUpdateCollection);
+    log.info("postUserContentUpdate: {}", updatesResult.getEntitiesForPreview());
```

Comment:
updatesResult.getEntitiesForPreview() could provide a huge amount of data, so from my point of view it is not good thing to log all that data as it will require a lot of cpu usage. If such kind of data needed it is better to log total amount of entities for preview.

Reply:
Good catch, thank you.